src/libraries/Microsoft.PowerFx.Core/Syntax/NamedFormula.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Parser;
+using Microsoft.PowerFx.Core.Syntax.Nodes;
+using Microsoft.PowerFx.Core.Utils;
+
+namespace Microsoft.PowerFx.Core.Syntax
+{
+    /// <summary>
+    /// This encapsulates a named formula: its original script, the parsed result, and any parse errors.
+    /// </summary>
+    internal sealed class NamedFormula
```

Comment:
should this be internal or public? It seems like it's being used from the powerfx side.

Reply:
ILanguageSettings and TexlNode are both internal though.

Reply:
ah, fair enough.